Documentation/gettingstarted/kind.rst

```diff
@@ -0,0 +1,244 @@
+and operation of Cilium in a multi-node Kubernetes cluster.
+
+Kind requires docker to be installed and running.
+
+Install Dependencies
+====================
+
+1. Install ``docker`` stable as described in: `Install Docker Engine <https://docs.docker.com/engine/install/>`_
+
+2. Install ``kubectl`` version >= v1.14.0 as described in the `Kubernetes Docs <https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_
+
+3. Install ``helm`` >= v3.0.3 per Helm documentation: `Installing Helm <https://helm.sh/docs/intro/install/>`_
+
+4. Install ``kind`` >= v0.7.0 per kind documentation: `Installation and Usage <https://kind.sigs.k8s.io/#installation-and-usage>`_
+
+Kind Configuration
+==================
+
+Kind doesn't use flags for configuration. Instead it uses YAML configuration that is very similar to Kubernetes.
+
+Create a ``kind-config.yaml`` file based on the following template. The template will create 3 node + 1 apiserver
+cluster with the latest version of kubernetes from when the kind release was created.
+
+.. code:: yaml
+
+    kind: Cluster
+    apiVersion: kind.x-k8s.io/v1alpha4
+    nodes:
+    - role: control-plane
+    - role: worker
+    - role: worker
+    - role: worker
+    networking:
+      disableDefaultCNI: true
+
+To change the version of kubernetes being run,  ``image`` has to be defined for each node. See
+the `Node Configration <https://kind.sigs.k8s.io/docs/user/configuration/#nodes>`_ documentation.
+
+Start Kind
+==========
+
+Pass the ``kind-config.yaml`` you created with the ``--config`` flag of kind.
+
+.. code:: bash
+
+    kind create cluster --config=kind-config.yaml
+
+This will add a ``kind-kind`` context to ``KUBECONFIG`` or if unset, ``${HOME}/.kube/config``
+
+.. code:: bash
+
+    kubectl cluster-info --context kind-kind
+
+.. _kind_install_cilium:
+
+Install Cilium
+==============
+
+.. include:: k8s-install-download-release.rst
+
+Install Cilium release via Helm:
+
+.. parsed-literal::
+
+   helm install cilium |CHART_RELEASE| \\
+      --namespace kube-system \\
+      --set global.nodeinit.enabled=true \\
+      --set global.kubeProxyReplacement=partial \\
+      --set global.hostServices.enabled=false \\
+      --set global.externalIPs.enabled=true \\
+      --set global.nodePort.enabled=true \\
+      --set global.hostPort.enabled=true
+
+.. include:: k8s-install-validate.rst
+.. include:: hubble-install.rst
+
+Next steps
+==========
+
+Now that you have a Kubernetes cluster with Cilium up and running, you can take
+a couple of next steps to explore various capabilities:
+
+* :ref:`gs_http`
+* :ref:`gs_dns`
+* :ref:`gs_cassandra`
+* :ref:`gs_kafka`
+
+Tips
+====
+
+Pre-load Cilium images into the kind cluster so each worker doesn't have to pull them.
+
+Run this before :ref:`kind_install_cilium`:
+
+.. parsed-literal::
+
+  docker pull cilium/cilium:|IMAGE_TAG|
+  kind load docker-image cilium/cilium:|IMAGE_TAG|
+
+And add ``--set global.pullPolicy=IfNotPresent`` to Helm install command.
```

Comment:
I wonder if this should be more prominent, eg integrated into the guide near the top with (optional but recommended).

Reply:
Integrated :)